src/heal.sol

```diff
@@ -116,14 +118,12 @@ contract Vow is DSNote {
     // Debt settlement
     function heal(uint wad) public note {
         require(wad <= Joy() && wad <= Woe());
-        require(int(mul(wad, ONE)) >= 0);
-        VatLike(vat).heal(bytes32(bytes20(address(this))), bytes32(bytes20(address(this))), int(mul(wad, ONE)));
+        VatLike(vat).heal(bytes32(bytes20(address(this))), bytes32(bytes20(address(this))), mul(wad, ONE));
```

Comment:
Oh so it's about not having the type conversion?

Reply:
Or rather, factoring out the type conversion to the multiplication function. I'm fine with omitting the second commit though.

Reply:
I slightly prefer the consistent implementation option FWIW.